### ISSUE TITLE
Function export csv/pdf/excel

### ISSUE THREAD

**Issue by leonfather** [ratiw/vuetable-2#91]
Hello ratiw, I have a question about export function, there is an implementation of csv/pdf/excel export ? I need to export my vuetable data?

**Comment by leonfather** [comment]
Ratiw i'm waiting your response?

**Comment by ratiw** [comment]
@leonfather No, as it in #39 and [here](https://github.com/ratiw/vue-table/issues/48)

Exporting to CSV should be pretty easy though. Try doing some research and work on that, you'll learn a lot.
